components/Tabs/examples/supplemental/TabBarViewControllerExampleSupplemental.m

```diff
@@ -46,8 +47,13 @@ - (void)drawRect:(CGRect)rect {
 @end
 
 @interface TBVCSampleViewController ()
+
 @property(nonatomic) MDCAppBarViewController *appBarViewController;
 @property(nonatomic) UILabel *titleLabel;
+@property(nonatomic) CGRect buttonFrame;
+@property(nonatomic) UIButton *button;
```

Comment:
Please make this `MDCButton *` since we cannot apply an `<MDCButtonScheming>` to a `UIButton`.

Reply:
Addressed in a46864d6635c5903a535e6436aed7fb783f084b3.